build.gradle

```diff
@@ -237,6 +272,10 @@ dependencies {
   testImplementation group: 'org.junit.platform', name: 'junit-platform-commons', version: versions.junitPlatform
   testImplementation group: 'org.junit.platform', name: 'junit-platform-engine', version: versions.junitPlatform
 
+  testCompile group: 'org.pitest', name: 'pitest', version: '1.5.1'
+  testCompile 'info.solidsoft.gradle.pitest:gradle-pitest-plugin:1.5.1'
+  testCompile 'org.codehaus.sonar-plugins:sonar-pitest-plugin:0.5'
```

Comment:
do those dependencies involve something? don't see any changes in tests so perhaps jsut allowing some access. but still - not convinced atm